client/app/telecom/telephony/billingAccount/orderAlias/nonGeographical/telecom-telephony-billing-account-orderAlias-nonGeographical.controller.js

```diff
@@ -78,6 +78,12 @@ angular.module('managerApp').controller('TelecomTelephonyAliasOrderNonGeographic
   this.changeQty = function () {
     this.form.pool = this.form.amount.value;
     this.form.numberType = this.form.amount.value === 1 ? this.form.numberType : 'common';
+    if (this.form.amount.value === 50 || this.form.amount.value === 100) {
```

Comment:
You could use `Array.prototype.includes()` in that case.

Does `50` and `100` could be turn into an angular constant?

```suggestion
    if ([50, 100].includes(this.form.amount.value)) {
```